ehr/resources/web/ehr/plugin/CollapsibleDataEntryPanel.js

```diff
@@ -14,6 +14,12 @@ Ext4.define('EHR.plugin.CollapsibleDataEntryPanel', {
             panel.collapsed = true;
         }
 
+        panel.onPanelDataChange = function() {
+            panel.expand();
+        }
+
+        panel.mon(panel, 'panelDataChange', panel.onPanelDataChange, {buffer: 500});
```

Comment:
I think the last argument will be interpreted at the scope and not the options. Can it be removed, or can we add a scope?

https://cdn.sencha.com/ext/gpl/4.2.1/docs/#!/api/Ext.util.Observable

`mon( item, ename, [fn], [scope], [options] ) `

Reply:
fixed thanks